### PR TITLE
Milestone 5: Desktop Version

### PR DESCRIPTION
**Pull request by Xghayor**
In this milestone, I completed:

1. Added CSS styles for Desktop Version

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -933,10 +949,10 @@ footer {
...
-    position: relative;
-    top: 0;
-    left: 0;
-    width: 100%;
+    position: relative !important;
+    width: 100% !important;
+    bottom: 0 !important;
+    left: 0 !important;
     border: 1px solid #212121;
   }
```
Comment (zieeco) [inline styles.css]: - Hey, great job so far, however, there are some issues you need to work on in order to have your project approved. Please, kindly see these suggestions as an opportunity to improve your development skill. Kindly fix all the issues that would be pointed out below:

   - Please, kindly add some spaces to the h3 to be close to the Figma design as much as you can.
   - According to the Microverse list for [HTML & CSS requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), You have overused the `!important` rule in your CSS style definitions which is not allowed and not a good practice, except you are using some CSS framework to add your styles. As this appears to be in multiple lines in your CSS style definitions, please, kindly ***REMOVE ALL THE `!important` RULE FROM YOUR CODEBASE*** and use another approach to add the desired rule you want to meet this part of the CSS requirement. :pray: 
   - 


*Figma design*
image


*Your design*
image

File: styles.css [hunk 2]
```diff
@@ -93,6 +93,11 @@ ul.navbar-items {
   list-style: none;
 }
 
+.contact-items ul li {
+  margin-bottom: 10px;
+  text-align: center;
+}
```
Comment (zieeco) [inline styles.css:99]: - Hey, let's add some spaces between each icon here on the desktop version and also, update the link to the last icon that is broken to improve our design.

*Figma desktop design*
image

*Your desktop version design*
image


- Let's also align the icons correctly in the mobile version as seen in the image below:

*Figma mobile version*
image

*Your design mobile version*
image

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Xghayor,



Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Nice adding media query ✔️
- Nice trying to implement the desktop design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions

- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 2 above)
Comment (Xghayor) [reply styles.css:99]: @zieeco Hi,
i would like to mention that in my screen it perfectly working, which branch did you downloaded to check?



please have a look on these screenshots, thank you

File: styles.css [hunk 3]
```diff
@@ -356,7 +370,8 @@ hr {
   font-size: 72px !important;
```
Comment (Efiamotu-1) [inline styles.css]: - As requested by the previous reviewer in this [review link](https://github.com/Xghayor/Portfolio/pull/6#discussion_r1231412715) you are yet to remove all of the instances where you have used the `!important` css property. 
- [ ] * According to the Microverse list for [HTML & CSS requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), You have overused the `!important` rule in your CSS style definitions which is not allowed and not a good practice, except you are using some CSS framework to add your styles. As this appears to be in multiple lines in your CSS style definitions, please, kindly _**REMOVE ALL THE `!important` RULE FROM YOUR CODEBASE**_ and use another approach to add the desired rule you want to meet this part of the CSS requirement.
- > please make this required changes and resubmit to be approved :+1: :sparkles:

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @Xghayor , 



Great work on making the changes requested by a previous reviewer 👏 Your Code is Awesome! 💪
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 3 above)
Comment (Xghayor) [reply styles.css]: @Efiamotu-1 
i was using only five, does it really exceed the limit?

**Review by Strangeal — APPROVED** [review]
Hi @Xghayor, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**